### PR TITLE
ci: Remove docker swarm installation

### LINKED ISSUE #4218: remove docker swarm installation as it is not being used

### PR DESCRIPTION
**Pull request by GabyCT**
This PR removes the docker swarm installation as currently kata 2.0
is not working with docker and we do not have any tests or features
where we are using this type of installation.

Fixes #4218

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Review by dborquez — APPROVED** [review]
LGTM, thank you @GabyCT

File: cmd/container-manager/manage_ctr_mgr.sh [hunk 1]
```diff
@@ -130,53 +128,12 @@ install_docker(){
 			sudo -E apt-get -y install "${pkg_name}"
 		elif [[ "$ID" =~ ^opensuse.*$ ]] || [ "$ID" == "sles" ]; then
 			sudo zypper removelock docker
-			sudo zypper -n  install "${pkg_name}" 
+			sudo zypper -n  install "${pkg_name}"
 			sudo zypper addlock docker
 		fi
-	elif [ "$tag" == "swarm" ]; then
```
Comment (wainersm) [inline cmd/container-manager/manage_ctr_mgr.sh:136]: So glad to see that clean up :)

**Review by wainersm — APPROVED** [review]

**Pull request merged**